router/pump.go

```diff
@@ -140,7 +140,7 @@ func (p *LogsPump) Run() error {
 		debug("pump.Run() event:", normalID(event.ID), event.Status)
 		switch event.Status {
 		case "start", "restart":
-			go p.pumpLogs(event, true, inactivityTimeout)
+			go p.pumpLogs(event, false, inactivityTimeout)
```

Comment:
Just saw your previous conversation. I think I'd like to keep backlog set to `true` on start for now. That at least then doesn't break the current expectation.